AEPMedia/Sources/MediaTracker.swift

```diff
@@ -14,30 +14,57 @@ import Foundation
 @objc(AEPMediaTracker)
 public protocol MediaTracker {
 
+    /// API to track the start of a viewing session.
+    /// - Parameter:
+    ///   - info: Dictionary created using `createMediaObject` API.
+    ///   - metadata: Dictionary containing the context data associated with the media session.
     @objc(trackSessionStart:metadata:)
     func trackSessionStart(info: [String: Any], metadata: [String: String]?)
 
+    /// API to track media play or resume after a previous pause.
     @objc(trackPlay)
     func trackPlay()
 
+    /// API to track media pause.
     @objc(trackPause)
     func trackPause()
 
+    /// API to track media complete.
     @objc(trackComplete)
     func trackComplete()
 
+    /// API to track the end of a viewing session.
+    /// This API must be called even if the user does not view the media to completion.
     @objc(trackSessionEnd)
     func trackSessionEnd()
 
+    /// API to track an error in media playback.
+    /// - Parameter:
+    ///   - errorId: `String` Identifier describing the error.
     @objc(trackError:)
     func trackError(errorId: String)
 
+    /// API to track media event.
+    /// - Parameter:
+    ///   - event: `MediaEvent` describing the event to track `ChapterStart`, `ChapterComplete`, `AdBreakStart`, `AdBreakComplete`, `AdStart`, `AdComplete`, `SeekStart`, `SeekComplete`, `BufferStart`,
+    ///   `BufferComplete`, `BitrateChange`.
+    ///   - info: `Dictionary` created using `createChapterObject`, `createAdBreakObject`, `createAdObject`, `createStateObject` API  with for `AdBreakStart`, `AdStart`, `ChapterStart`, `StateStart` and `StateEnd` events respectively. Pass nil for other events.
```

Comment:
Matching the order
- info: `Dictionary` created using `createChapterObject`, `createAdBreakObject`, `createAdObject`, `createStateObject` API with for `ChapterStart`, `AdBreakStart`, `AdStart`, `StateStart` and `StateEnd` events respectively.